### ISSUE TITLE
KeyError: 'result' when pokestop is spinning

### ISSUE THREAD

**Issue by hnq90** [PokemonGoF/PokemonGo-Bot#594]
### Expected Behavior
Got items from pokestop

### Actual Behavior
```
[2016-07-24 06:24:56] [#] Finished walking
[2016-07-24 06:24:56] [#] Arrived at Pokestop
2016-07-24 06:24:57 [#] Now at Pokestop: Kids Pirate Ship - Spinning...
Traceback (most recent call last):
  File "pokecli.py", line 219, in <module>
    main()
  File "pokecli.py", line 210, in main
    bot.take_step()
  File "/home/huynq/PokemonGo-Bot/pokemongo_bot/__init__.py", line 36, in take_step
    self.stepper.take_step()
  File "/home/huynq/PokemonGo-Bot/pokemongo_bot/stepper.py", line 67, in take_step
    self._work_at_position(position[0], position[1], position[2], True)
  File "/home/huynq/PokemonGo-Bot/pokemongo_bot/stepper.py", line 140, in _work_at_position
    self.bot.work_on_cell(cell, position, pokemon_only)
  File "/home/huynq/PokemonGo-Bot/pokemongo_bot/__init__.py", line 88, in work_on_cell
    hack_chain = worker.work()
  File "/home/huynq/PokemonGo-Bot/pokemongo_bot/cell_workers/seen_fort_worker.py", line 51, in work
    if spin_details['result'] == 1:
KeyError: 'result'
```

### Steps to Reproduce
Error occurred randomly when farming.

### Other Information
OS: Ubuntu 14.04
Git Commit: 1ce23d70afd006c165ab11d9c3ac2e2c984ed568

**Comment by surfaace** [comment]
Please clarify if issue occures with altest dev branch

**Comment by sonontse** [comment]
`Traceback (most recent call last):
  File "pokecli.py", line 222, in <module>
    main()
  File "pokecli.py", line 213, in main
    bot.take_step()
  File "/var/www/html/public/123/pokemongo_bot/__init__.py", line 37, in take_step
    self.stepper.take_step()
  File "/var/www/html/public/123/pokemongo_bot/stepper.py", line 62, in take_step
    self._work_at_position(position[0], position[1], position[2], True)
  File "/var/www/html/public/123/pokemongo_bot/stepper.py", line 139, in _work_at_position
    self.bot.work_on_cell(cell, position, pokemon_only)
  File "/var/www/html/public/123/pokemongo_bot/__init__.py", line 98, in work_on_cell
    hack_chain = worker.work()
  File "/var/www/html/public/123/pokemongo_bot/cell_workers/seen_fort_worker.py", line 51, in work
    if spin_details['result'] == 1:
KeyError: 'result'`

Just had this exact error come up with the latest dev build while getting results from a pokestop.  Ubuntu 14.04.  Git Commit 55adefe

**Comment by lollan** [comment]
Could this be from a ban?  I am seeing this on 2 of my accounts.   I tried using a third account and I can spin at pokestops without problem.

**Comment by buettner123** [comment]
not sure if this points to a softban. When I play the game on my phone I randomly get "try again later" on one Pokestop but on the next one a few steps ahead everything works fine. After this error the pokestop is not on cooldown and is displayed like I could spin it again.

**Comment by buettner123** [comment]
Seems to be fixed with PR #974 and is duplicated by #922

**Comment by TheGoldenXY** [comment]
This issue has been fixed in newest branch.

/closed
